src/components/helpers/pipelineParameterMapping.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
```

Comment:
You should avoid using React (jsx) syntax in helpers if possible. JSX should be used in Components only. If the messages are really kept here (i.e., if you will not meet my previous suggestion), you should use react-inlt defineMessages function instead of <FormattedMessage> component.